daemon/list.go

```diff
@@ -197,14 +201,11 @@ func (daemon *Daemon) Containers(job *engine.Job) error {
 	for _, container := range daemon.List() {
 		if err := writeCont(container); err != nil {
 			if err != errLast {
-				return err
+				return nil, err
 			}
 			break
 		}
 	}
 	sort.Sort(sort.Reverse(ByCreated(containers)))
```

Comment:
hmmm can you double check... I agree daemon.List is sorted, but this wants it reversed.